src/main/resources/flows-config.yaml

```diff
@@ -536,10 +536,18 @@ flow:
   docUploadType:
     beforeSaveAction: AssociateDocTypeLabel
     nextScreens:
-      - name: docUploadReview
+      - name: docUploadReviewTypes
         condition: ContinueAssociateDocType
       - name: docUpload
-  docUploadReview:
+  docUploadOwner:
```

Comment:
Tiny, non blocking. This reads easier when the sections are in order. I'd expect docUploadReviewTypes to be in line 542 and then docUploadOwner to be listed afterward.